vcr/verifier/interface.go

```diff
@@ -48,8 +48,8 @@ type Verifier interface {
 	RegisterRevocation(revocation credential.Revocation) error
 
 	// VerifyVP verifies the given Verifiable Presentation. If successful, it returns the credentials within the presentation.
-	// If verifyVCs is true, it will also verify the credentials inside the VP, checking their correctness, signature and trust status.
-	VerifyVP(presentation vc.VerifiablePresentation, verifyVCs bool, validAt *time.Time) ([]vc.VerifiableCredential, error)
+	// If verifyVCs is true, it will also verify the credentials inside the VP, checking their correctness, signature and trust status (unless verifyVCsTrusted is false).
+	VerifyVP(presentation vc.VerifiablePresentation, verifyVCs bool, verifyVCsTrusted bool, validAt *time.Time) ([]vc.VerifiableCredential, error)
```

Comment:
As far as I can tell `verifyVCsTrusted ==  !allowUntrusted` (from `Verify()` in the same interface)? If so, I'd prefer to use `allowUntrusted` here too to make it easier to understand. (partially because I don't find `verifyVCsTrusted` very clear)

Reply:
`allowUntrusted` is also used for this in `Finder.Search`

Reply:
I followed the `verify` pattern from `verifyVCs`, so at least to make it consistent within this function signature.

I think "allowUntrusted" was the wrong choice, since the default (true) should be "verifyTrusted" imo

Reply:
The default value for a bool is false, so when adding `allowUntrusted` to a struct it would actually default to the safe option.

Reply:
Ok I'll stick to `allowUntrustedVCs` then